### PR TITLE
Fix typos, formatting errors, etc.

### PR DESCRIPTION
**Pull request by eschan145**
Changed several comments to capitalization and added more docs.

* _Counter-clock-wise_ → **counterclockwise**
* _Over-ride_ → **override**

### CODE REVIEW

**Comment by eschan145** [comment]
I might add more commits to this, so don't merge immediately.

**Comment by pvcraven** [comment]
> I might add more commits to this, so don't merge immediately.

Let us know when you are ready.

**Comment by eschan145** [comment]
@pvcraven should I change all of the line widths to 80 characters? It's inconsistent all over the place. Some are wrapped to 100 chars, some to 80.

File: arcade/text.py
Comment (einarf) [inline arcade/text.py:239]: It's important to communicate if the property is read/write. That's the reason for the "get or set" part.

This applies to most or all propreties here.

**Review by einarf — CHANGES_REQUESTED** [review]

**Comment by einarf** [comment]
It would probably be better/easier to make smaller PRs for these things? Something that could just be looked at briefly and merged within the same day or week. Some of these comments are already changed in other branches for example.. or the code have moved or changed in other ways.

File: arcade/text.py
Comment (eschan145) [reply arcade/text.py:239]: It's kind of repetitive, though

Comment (eschan145) [reply arcade/text.py:239]: If it's read only, it's usually stated that `Read-only` to clarify.

Comment (einarf) [reply arcade/text.py:239]: It's definitely repetative, but it's explicit not creating doubt. We've had so many examples of users not understanding the properties can actually be set. It's immediately visible in your IDE, python `help()` and in sphinx.

**Comment by eschan145** [comment]
@einarf I think I'm ready to merge. There is one conflict in `window_commands`. My pull request should override it.

**Comment by einarf** [comment]
There are lots of other cross branch conflicts due to this PR being so old and way too large.

Still, there are some good things in here. I suspect this PR just needs to be manually applied after the texture-revamp branch is merged.

**Pull request closed without merging**
